types/webpack-bundle-analyzer/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-outside-dependencies": false
+    }
```

Comment:
```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```

Reply:
this is outcome of webpack ecosystem problems. Please try:
```ts
--- a/types/webpack-bundle-analyzer/tsconfig.json
+++ b/types/webpack-bundle-analyzer/tsconfig.json
@@ -13,6 +13,9 @@
             "../"
         ],
         "paths": {
+            "buffer": [
+                "./node_modules/@types/node/buffer"
+            ],
             "webpack": [
                 "./node_modules/webpack"
             ],
diff --git a/types/webpack-bundle-analyzer/tslint.json b/types/webpack-bundle-analyzer/tslint.json
index d53922ab74..2efa283002 100644
--- a/types/webpack-bundle-analyzer/tslint.json
+++ b/types/webpack-bundle-analyzer/tslint.json
@@ -1,6 +1,3 @@
 {
-    "extends": "@definitelytyped/dtslint/dt.json",
-    "rules": {
-        "no-outside-dependencies": false
-    }
+    "extends": "@definitelytyped/dtslint/dt.json"
 }
 ```
 The webpack ecosystem could fix this problem by staring using 'node:buffer' in imports, instead of only 'buffer', as there is 'buffer' module on NPM.

Reply:
@peterblazejewicz Tried your suggestion, didn't work 😕 
I also don't see any other package that does this, it seems like every other package that uses native nodejs modules has to add that lint rule.

Reply:
the lint error is proper one, that's not a solution to silent one. Give me a sec to review another option (except of Webpack adopting namespaced imports internally)

Reply:
OK, I cannot make it working (I was thinkging about using overrides via augentation, the problem is within node_modules/webpack/types.d.ts` when used in context of DT types). I've added comments to restore your changes.
Sorry for dragging this review due to this problem. I'll post an issue on webpack repo

Reply:
That's perfectly fine, I wasn't completely happy with that solution as well, but it seems like there's no choice sadly 🥲